pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/documentation/CommentContentRule.java

```diff
@@ -35,15 +36,14 @@
     private List<String> originalBadWords;
     private List<String> currentBadWords;
 
-    // FIXME need some better defaults (or none?)
-    private static final String[] BAD_WORDS = {"idiot", "jerk" };
-
     // ignored when property above == True
     public static final BooleanProperty CASE_SENSITIVE_DESCRIPTOR = new BooleanProperty("caseSensitive",
             "Case sensitive", false, 2.0f);
 
-    public static final StringMultiProperty DISSALLOWED_TERMS_DESCRIPTOR = new StringMultiProperty("disallowedTerms",
-            "Illegal terms or phrases", BAD_WORDS, 3.0f, '|');
+    public static final PropertyDescriptor<List<String>> DISSALLOWED_TERMS_DESCRIPTOR =
+            stringListProperty("disallowedTerms")
+                    .desc("Illegal terms or phrases")
+                    .defaultValues("idiot", "jerk").build(); // TODO make blank property? or add more defaults?
```

Comment:
`delim(",")`?

Reply:
See the last arg of the constructor, which is the delimiter '|', equal to the default delimiter